### ISSUE TITLE
Infinite Searching Timers

### ISSUE THREAD

**Issue by Cringeworthy23** [arc298/instagram-scraper#819]
**Describe the bug**
While running, the script pauses while searching a user for posts infinitely. 
**To Reproduce**
1. instagram-scraper  -u **** -p **** --cookiejar cookies.jar --latest --filename ig

**Expected behavior**
Expected to continue with no problems

**System (please complete the following information):**
 - OS: Windows 11

**Comment by Raymon360** [comment]
> **Describe the bug** While running, the script pauses while searching a user for posts infinitely. Stuck on: Searching ***** for posts: 0 media [00:00, ? media/s] **To Reproduce** Steps to reproduce the behavior:
> 
> 1. instagram-scraper  -u **** -p **** --cookiejar cookies.jar --latest --filename ig
> 
> **Expected behavior** Expected to continue with no problems
> 
> **System (please complete the following information):**
> 
> * OS: Windows 11

yes I had a similar issue with HTTPError Error output ('400 Client Error: Bad Request for url: https://i.instagram.com/api/v1/media/2403017234223123592/info/')
What should I do to solve this problem?

**Comment by Cringeworthy23** [comment]
@nns33213 Is it too much to scrape daily on an account? Or should I rotate accounts?

**Comment by Cringeworthy23** [comment]
@nns33213 I got banned from Instagram, but before, I was scraping around 100 profiles, most of them rarely active, simply to download any new content when my friends post it.
